src/test/java/org/reaktivity/ry/internal/RyMainTest.java

```diff
@@ -15,26 +15,106 @@
  */
 package org.reaktivity.ry.internal;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.reaktivity.ry.internal.RyTestCommandSpi.TEST_ARGUMENT;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.jar.JarEntry;
+import java.util.jar.JarOutputStream;
+
+import javax.naming.spi.Resolver;
+
+import org.apache.ivy.Ivy;
+import org.apache.ivy.ant.AntWorkspaceResolver;
+import org.apache.ivy.ant.AntWorkspaceResolver.WorkspaceArtifact;
+import org.apache.ivy.core.install.InstallOptions;
+import org.apache.ivy.core.module.id.ModuleRevisionId;
+import org.apache.ivy.core.report.ResolveReport;
+import org.apache.ivy.core.settings.IvySettings;
+import org.apache.ivy.plugins.resolver.FileSystemResolver;
+import org.apache.ivy.util.DefaultMessageLogger;
+import org.apache.ivy.util.Message;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.reaktivity.ry.RyCommandSpi;
 
 public class RyMainTest
 {
+    @BeforeAll
```

Comment:
Good decision to use `@BeforeAll` instead of `@Before`, it is slightly more efficient in this case, where you have 2 separate tests to run, and the resulting output of `setup()` is immutable, so safely shared by both tests without side effects.

Reply:
Lol, I'd love to take credit, but there was no `@Before`. Only `@BeforeAll` or `@BeforeEach` were on offer. Or is `@BeforeEach` the same as `@Before`?